src/lib/constants/EnvVariablesConstants.ts

```diff
@@ -0,0 +1,3 @@
+export enum EnvVariablesConstants {
```

Comment:
NIT:
What do you think about `src/lib/constants/EnvVariables.ts` instead? That is, changing the file name from `EnvVariableContants.ts` to `EnvVariables.ts`. (And the same for `PathConstants.ts` in te same folder)